src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -283,15 +282,22 @@ int mongoSubCacheItemInsert
   //
   // 01. Check incoming parameters
   //
-  if ((subscriptionId == NULL) || (subscriptionId[0] == 0))
+  // FIXME PR: check the commented lines. They correspond to conflict solving
+  //
+  //if ((subscriptionId == NULL) || (subscriptionId[0] == 0))
+  if (subscriptionId.empty())
   {
     return -1;
   }
 
-  if ((servicePath == NULL) || (servicePath[0] == 0))
-  {
-    return -2;
-  }
+  // FIXME PR: empty servicepath is legal, in means "default".
+  // Thuss this checking is not meaninfull anymore and -2 removed
+  // from the list (and probably offset -3, -4, etc. a step)
+  //if ((servicePath == NULL) || (servicePath[0] == 0))
+  //if (servicePath.empty())
+  //{
+  //  return -2;
+  //}
```

Comment:
empty servicepath may be legal in the "beginning of the request", but here ... not so sure.
I mean, if no service path is given (or empty service path), then we "force" the default service path on the request.
But in this point of the execution, if service path is empty, something has gone wrong, I believe.
Then again, checking for this may be a little paranoid, except for when dealing with pointers as NULL pointers may crash the system if used. 

OK to remove the entire check